Generate CRUD by default for SDL and Services to mitigate issues in GraphQL used in cells (#4785)
redwoodjs/redwood: docs/docs/cli-commands.md

| Arguments & Options  | Description                                                                          |
| -------------------- | ------------------------------------------------------------------------------------ |
| `model`              | Model to generate the sdl for                                                        |

> | `--crud`             | Set to `false` if you do not want to generate mutations                              |

Reviewer: ```suggestion
| `--crud`             | Set to `false`, or use `--no-crud`, if you do not want to generate mutations                              |
```

Done in 1.04s.
```

> The generated sdl defines a corresponding type, query, create/update inputs, and any mutations. To prevent defining mutations, add the `--crud=false` option.

Reviewer: ```suggestion
The generated sdl defines a corresponding type, query, create/update inputs, and any mutations. To prevent defining mutations, add the `--no-crud` option.
```

    email: String
    name: String
  }

  type Mutation {
    createUser(input: CreateUserInput!): User! @requireAuth
    updateUser(id: Int!, input: UpdateUserInput!): User! @requireAuth
    deleteUser(id: Int!): User! @requireAuth
  }
`
```

> The services file fulfills the query with . If the `--crud=false` option is added, this file will be less complex.

Reviewer: ```suggestion
The services file fulfills the query. If the `--no-crud` option is added, this file will be less complex.
```

You added ` with ` to the first sentence here. I don't understand how to read that to make any sense out of it. The extra space after "with " makes me think you forgot some word, or had a thought you didn't finish.
